### PR TITLE
[zh-CN] initial translation of Notification.tag

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

initial translation of Notification.tag

### Motivation

initial translation of Notification.tag

### Additional details

https://github.com/mdn/content/blob/main/files/en-us/web/api/notification/tag/index.md?plain=1

### CODE REVIEW

File: files/zh-cn/web/api/notification/tag/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+通知的标签的构想是多个通知可以共享同一个标签，将它们链接在一起。然后可以通过编程方式将一个通知替换为另一个通知，以避免用户的屏幕被大量类似的通知填满。
+
+## 值
+
+一个字符串。
+
+## 示例
+
+我们的[使用通知 API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API#dealing_with_repeated_notifications) 文章有一个很好的标签属性的使用示例。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notification/tag/index.md:18]: 此链接hash在英文文档已失效

File: files/zh-cn/web/api/notification/tag/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Notification：tag 属性
+slug: Web/API/Notification/tag
+---
+
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
+
+{{domxref("Notification")}} 接口的 **`tag`** 只读属性表示通知的识别标签，在 {{domxref("Notification. Notification","Notification()")}} 构造函数的 `tag` 选项中指定。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/notification/tag/index.md:8]:
```suggestion
{{domxref("Notification")}} 接口的 **`tag`** 只读属性表示通知的识别标签。此属性通过 {{domxref("Notification.Notification","Notification()")}} 构造函数的 `tag` 选项中指定。
```

File: files/zh-cn/web/api/notification/tag/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/web/api/notification/tag/index.md:18]: PR: https://github.com/mdn/content/pull/27546

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

About the problem in link, I've raised https://github.com/mdn/content/pull/27546 to propose a posible fix.

File: files/zh-cn/web/api/notification/tag/index.md (hunk 1 above)
Comment (skyclouds2001) [reply files/zh-cn/web/api/notification/tag/index.md:18]: Got it

**Comment by skyclouds2001** [comment]
change to draft, as is waiting for updating for other pull request

File: files/zh-cn/web/api/notification/tag/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/web/api/notification/tag/index.md:18]:
```suggestion
我们的[使用通知 API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API#replacing_existing_notifications) 文章有一个很好的标签属性的使用示例。
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**